server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java

```diff
@@ -139,6 +139,11 @@ public CeMainThread() {
     public void run() {
       boolean startupSuccessful = attemptStartup();
       this.started = true;
+      try {
+        Thread.sleep(2000);
+      } catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2142)